CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ This project adheres to the Node default version scheme, meaning It's safe to us
 The standard for the caret can [be found here](https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004).
 Breaking changes result in a different major. UI changes that might break customizations on top of the sdk, will be treated as breaking changes too.
 
+## [0.6.0]
+### Added
+- API: `useWebcam` options added to facial and document capture step
+- API: Possible to change options at runtime by calling `setOptions()` on Object returned by `Onfido.init()`
+### Changed
+- Internal: isDesktop detection is now based on WebRTC isMobile detection
+- Internal: improved Webcam Detection, looking at more than **getUserMedia**, uses WebRTC for that
```

Comment:
The changelogs are not very informative. Have you also updated the documentation?